front/app/component-library/components/Tooltip/index.tsx

```diff
@@ -69,45 +69,45 @@ const Tooltip = ({
   }, [activeElement, isFocused]);
 
   return (
-    <Tippy
-      key={key}
-      plugins={[
-        {
-          name: 'hideOnEsc',
-          defaultValue: true,
-          fn({ hide }) {
-            function onKeyDown(event: KeyboardEvent) {
-              if (event.key === 'Escape') {
-                hide();
+    <Box as="span" id={tooltipId.current} w={width || 'fit-content'}>
+      <Tippy
```

Comment:
In this file, I just moved the `<Box as="span" id={tooltipId.current} w={width || 'fit-content'}> `outisde of the `<Tippy>` component.

I looked at many tooltips in our platform after making this change, and they actually all seem to work correctly still. Now the `aria-expanded` attribute also appears on the correct `<Button>` child component when Tippy is used.